.jenkins/agent/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM quay.io/openshift/origin-jenkins-agent-base:latest
 RUN curl -LO "https://github.com/operator-framework/operator-sdk/releases/download/v0.19.4/operator-sdk-v0.19.4-x86_64-linux-gnu" && \
     chmod +x operator-sdk-v0.19.4-x86_64-linux-gnu && mv operator-sdk-v0.19.4-x86_64-linux-gnu /usr/local/bin/operator-sdk
-RUN dnf install -y ansible golang && \
+RUN dnf install -y ansible golang python38 && \
     dnf groupinstall -y "Development Tools" -y && \
-    ansible-galaxy collection install community.kubernetes && \
-    python -m pip install openshift kubernetes
+    alternatives --set python /usr/bin/python3.8 && \
+    python -m pip install openshift kubernetes "ansible-core~=2.12" && \
+    ansible-galaxy collection install -f kubernetes.core community.general
```

Comment:
community.kubernetes has been replaced with kubernetes.core, and the `make` module we use in loki building (even if it's turned off) is now part of community.general which must be installed explicitly when using ansible-core